docs/pages/management/operations/backup-restore.mdx

```diff
@@ -137,6 +133,18 @@ $ sudo teleport start --config fresh.yaml --bootstrap state.yaml
 $ tctl --config fresh.yaml get all
 # <your state here>
 ```
+<Notice>
+The `tctl get all` must be run on a Teleport Auth Service host 
+or existing User records will cause a permission error.  This is due
+to User secrets only being available from a a direct `tctl` connection
+on a Teleport Auth Service machine. As a workaround, retrieve resources
+individually with the `--with-secrets` parameter except for users. Local user secrets
+including passwords would need to be reset after creating without secrets.
```

Comment:
```suggestion
<Notice type="danger">

The `tctl get all` command must be run on a Teleport Auth Service host or existing user records will cause a permission error.

</Notice>

Teleport user secrets are only available when running `tctl` commands
on a Teleport Auth Service machine. 

As a workaround, retrieve resources
individually with the `--with-secrets` parameter except for users. Local user secrets
including passwords will need to be reset after being created without secrets.
```

- Making the `Notice` smaller and more impactful using the `danger` type. Otherwise the visual qualities of the `Notice` become noise.
- Minor grammar/readability tweaks.